src/main/java/com/kk/strong/model/User.java

```diff
@@ -18,10 +18,27 @@ public class User {
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
     private Long id;
-    private String name;
     private String username;
     private String password;
 
+    private String name;
+    private GenderType gender;
+    private int age;
+    private int height;
```

Comment:
you are slowly seeing that the login data is mixed with business logic data, if that will be the case I would separate this two:
1. User for all the login, register, roles stuff
2. GymUser or any other name for all the business logic stuff